cluster/charts/rook-ceph/values.yaml

```diff
@@ -67,6 +67,8 @@ csi:
   # CSI Rbd plugin daemonset update strategy, supported values are OnDelete and RollingUpdate.
   # Default value is RollingUpdate.
   #cephFSPluginUpdateStrategy: OnDelete
+  # Allow starting unsupported ceph-csi images
```

Comment:
`%s/images/image` in yaml files as we are dealing with a single CSI image

Reply:
need to document this in helm document https://github.com/rook/rook/blob/master/Documentation/helm-operator.md#configuration

Reply:
Will add the flag to the doc.